### PR TITLE
You can't put more than 2 guns(pistol, revolver) in bags anymore

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

You can't put more than 2 pistols in bags anymore.

Pistols can be used as a secondary, or you can reload them pretty efficiently with the pistol belt.

## Why It's Good For The Game

https://media.discordapp.net/attachments/504094319075131402/881174890210820196/sins.PNG?width=874&height=670

Pistols loadout have:
-Incredible DPS. You can kill ancient runner in one sec with tp 23
-Amazing sunder
-Instant reload. Taking a two new pistols from a combat back pack takes half a sec
-Insane speed. Because zero slowdown, no need to wield
-Basically immune to stun. You don't have to wield, you can just take new pistols

And the compromise for that is:
- It's long to make your l... Wait, no longer the case, you can just use loadout vendor
- You have to come to fob to reload quite often. 

That's it.

Pistols in bag are an issue since the beginning, it's time to kill that meme

## Changelog
:cl:
balance: You can't put more than 2 guns in bags anymore.
/:cl:

### CODE REVIEW

**Comment by Eskjjlj** [comment]
That would make the game too easy for us xeno players.

**Comment by SpaceLoveSs13** [comment]
Much needed balance nerf. It is completely surprising this thing that one can just press E and spam 20 times and outrun xenoes with can kill better and more than normal rifles. Spamming is just unacceptable and proof of bad balance. This also doesn't help the fact that marines can just spam vend multiple bags of same loadout bearing multiple loaded pistols.

Good PR.

**Comment by Pariah919** [comment]
this doesn't fix anything and is a blanket nerf to people who use pistols as sidearms in their bag which is common enough
if you want to fix the issue make it g8 belt, dummy

**Comment by Ragecrabbo** [comment]
Why not add delay to drawing guns out of bags or something? Making them unable to be put in bags at all seems a bit too extreme of a nerf for sidearm users.

**Comment by lbnesquik** [comment]
No! Maybe put a LIMIT on the number of pistols in backpack but do not forbid the carrying of pistols around. I am NOT wasting my belt or pouch slot for a backup weapon.

**Comment by mathis650** [comment]
unbased

**Comment by SilencedMP5A5** [comment]
maybe nerf the TP23 instead of making a funny gimmick change

File: code/game/objects/items/storage/backpack.dm [hunk 1]
```diff
@@ -12,6 +12,9 @@
 	storage_slots = null
 	max_storage_space = 24
 	access_delay = 1.5 SECONDS
+	storage_type_limits = list(
+		/obj/item/weapon/gun = 2,
```
Comment (Pariah919) [inline code/game/objects/items/storage/backpack.dm:16]:
```suggestion
		/obj/item/weapon/gun/pistol = 2,
		/obj/item/weapon/gun/pistol/standard_pocketpistol,
```

**Review by TiviPlus — APPROVED** [review]

**Comment by Derrick2Electricboogaloo** [comment]
If only I had enough handguns to show you how many I could fit in a backpack :)
